### PR TITLE
fix: remove portal from header

### PR DESCRIPTION
**Pull request by vasyl-ivanchuk**
# What ❔

- Remove Portal link from the top menu
- Do not show an error reason for the failed transaction if it's empty
- Docker compose starts services in production and not in development mode
- Add zkVM filter option for solc compilers on the verification page

## Why ❔

To fix bug reports / implement feature requests.

## Checklist

- [X] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [X] Tests for the changes have been added / updated.

### CODE REVIEW

File: packages/app/src/components/form/FormItem.vue [hunk 1]
```diff
@@ -42,7 +42,7 @@ defineProps({
 }
 .label-inline-block {
   .form-item-label {
-    @apply inline-block;
+    @apply float-left;
```
Comment (Romsters) [inline packages/app/src/components/form/FormItem.vue:45]: Are you sure it won't break any existing use cases?

File: packages/app/src/components/common/CheckBoxInput.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+export default {
+  inheritAttrs: false,
+};
+</script>
+
+<script setup lang="ts">
+import { computed } from "vue";
+const props = defineProps({
+  modelValue: {
+    type: [Boolean],
```
Comment (Romsters) [inline packages/app/src/components/common/CheckBoxInput.vue:18]: Why not just `type: Boolean` ?

Comment (vasyl-ivanchuk) [reply packages/app/src/components/common/CheckBoxInput.vue]: Good point, fixed.

File: packages/app/src/components/form/FormItem.vue (hunk 1 above)
Comment (vasyl-ivanchuk) [reply packages/app/src/components/form/FormItem.vue:45]: I don't think so. This class used in two places and I checked them both.

**Review by Romsters — APPROVED** [review]

**Pull request merged**